### PR TITLE
Encapsulation reactive variables to optimize running time and avoid unnecessary queries

### PR DESCRIPTION
**Pull request by Nguyen379**

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -19,118 +19,228 @@ server <- function(input, output, session) {
...
+                             table = "clinical",
+                             dataset = input$dataset)
+    }
+    return(dbGetQuery(database, query))
+  })
+
+  # Interactive query data
+  queryData <- reactive({
+    # default value
+    clinical <- clinicalData()
```
Comment (chrisamiller) [inline server.R]: Since in some cases, this result will not be from the clinical table, would another variable/function name make sense?

**Review by chrisamiller — COMMENTED** [review]
Just one minor comment about variable naming. If it's an easy fix, then I'd say clean it up (let me know if you want to talk about alternative names). If not, then I think we can merge this.

File: server.R (hunk 1 above)
Comment (Nguyen379) [reply server.R]: yeah sure. Do you have something in mind?

Comment (chrisamiller) [reply server.R]: perhaps "metadata" makes sense, since it's not the data being plotted (expression), it's info about the data being plotted

**Pull request merged**